src/pages/App.tsx

```diff
@@ -200,6 +200,12 @@ export default function App() {
     return null
   }
 
+  const blockedPaths = document.querySelector('meta[name="x:blocked-paths"]')?.getAttribute('content')?.split(',')
+  const shouldBlockPath = blockedPaths?.includes(pathname) ?? false
+  if (shouldBlockPath && pathname !== '/swap') {
```

Comment:
idk, I feel like you should include some weird snarky comment about how you can't block /swap:

```suggestion
  // Uniswap, the unstoppable swappable
  if (shouldBlockPath && pathname !== '/swap') {
```